<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>642</width>
    <height>474</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>나눔고딕</family>
   </font>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>621</width>
      <height>151</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QGroupBox" name="groupBox">
       <property name="title">
        <string>정보입력</string>
       </property>
       <widget class="QLabel" name="label">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>70</y>
          <width>60</width>
          <height>12</height>
         </rect>
        </property>
        <property name="text">
         <string>이       름 : </string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
       <widget class="QLineEdit" name="txtName">
        <property name="geometry">
         <rect>
          <x>90</x>
          <y>70</y>
          <width>191</width>
          <height>20</height>
         </rect>
        </property>
        <property name="inputMethodHints">
         <set>Qt::ImhDigitsOnly|Qt::ImhLatinOnly</set>
        </property>
        <property name="inputMask">
         <string/>
        </property>
        <property name="placeholderText">
         <string/>
        </property>
       </widget>
       <widget class="QLabel" name="label_3">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>95</y>
          <width>60</width>
          <height>12</height>
         </rect>
        </property>
        <property name="text">
         <string>전화번호 : </string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
       <widget class="QLineEdit" name="txtPhone">
        <property name="geometry">
         <rect>
          <x>90</x>
          <y>95</y>
          <width>191</width>
          <height>20</height>
         </rect>
        </property>
        <property name="inputMethodHints">
         <set>Qt::ImhFormattedNumbersOnly</set>
        </property>
       </widget>
       <widget class="QLineEdit" name="txtCarNum">
        <property name="geometry">
         <rect>
          <x>90</x>
          <y>20</y>
          <width>191</width>
          <height>20</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="label_4">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>20</y>
          <width>60</width>
          <height>12</height>
         </rect>
        </property>
        <property name="text">
         <string>차 번호 :</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
       <widget class="QLineEdit" name="txtSubPhone">
        <property name="geometry">
         <rect>
          <x>90</x>
          <y>120</y>
          <width>191</width>
          <height>20</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="label_5">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>120</y>
          <width>71</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>비상 연락망 :</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
       <widget class="QPushButton" name="btnNew">
        <property name="geometry">
         <rect>
          <x>450</x>
          <y>100</y>
          <width>80</width>
          <height>30</height>
         </rect>
        </property>
        <property name="text">
         <string>신규</string>
        </property>
       </widget>
       <widget class="QPushButton" name="btnSave">
        <property name="geometry">
         <rect>
          <x>530</x>
          <y>100</y>
          <width>80</width>
          <height>30</height>
         </rect>
        </property>
        <property name="text">
         <string>저장</string>
        </property>
       </widget>
       <widget class="QPushButton" name="btnDel">
        <property name="geometry">
         <rect>
          <x>529</x>
          <y>68</y>
          <width>80</width>
          <height>30</height>
         </rect>
        </property>
        <property name="accessibleDescription">
         <string/>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color:rgb(255,50,50);
color:rgb(255,255,255)</string>
        </property>
        <property name="text">
         <string>삭제</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="txtCarName">
        <property name="geometry">
         <rect>
          <x>90</x>
          <y>45</y>
          <width>191</width>
          <height>20</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="label_9">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>45</y>
          <width>60</width>
          <height>12</height>
         </rect>
        </property>
        <property name="text">
         <string>차종 :</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>169</y>
      <width>621</width>
      <height>281</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QTableWidget" name="tblcarinfo"/>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QStatusBar" name="stbCurrent"/>
 </widget>
 <resources/>
 <connections/>
</ui>
